docs/DeveloperGuide.md

```diff
@@ -643,80 +643,183 @@ otherwise)
 
 ---
 
-**Use Case: UC01 - Add Animal**
+**Use Case: UC01 - Help on Animal Commands**
 
 **MSS:**
-1. User chooses to add an animal.
-2. AnimalCatalog prompts for the animal details.
-3. User provides the necessary animal details.
-4. AnimalCatalog validates the details.
-5. AnimalCatalog adds the pet to the shelter database and displays the success message.
-   Use case ends.
+1. User requests help on a specific command or a list of commands related to animal handling.
+2. AnimalCatalog processes the request and identifies the command or commands in question.
+3. AnimalCatalog retrieves the usage instructions for the command(s).
```

Comment:
IIRC, a use case describes only the externally visible behavior, not internal details, of a system i.e. should minimize details that are not part of the interaction between the user and the system. Maybe line 651 isn't needed or could combine with line 650